src/applications/ivc-champva/10-7959C/helpers/utilities.js

```diff
@@ -13,7 +13,7 @@ export function nameWording(formData, isPosessive = true, cap = true) {
     retVal = isPosessive ? 'your' : 'you';
   } else {
     // Concatenate all parts of applicant's name (first, middle, etc...)
-    retVal = Object.values(formData.applicantName)
+    retVal = Object.values(formData?.applicantName || {})
```

Comment:
Fixes bug where the review page would break if `applicantName` was temporarily undefined (as it sometimes is on initial load)